webots_ros2_core/webots_ros2_core/utils.py

```diff
@@ -56,6 +58,7 @@ def get_webots_home(version=get_required_webots_version()):
     # then using the 'which' command
     try:
         path = os.path.split(os.path.abspath(subprocess.check_output(['which', 'webots'])))[0]
+        path = path.decode('utf-8')
```

Comment:
This lines is the actual fix for `Can't mix strings and bytes in path components`